torch/export/dynamic_shapes.py

```diff
@@ -29,13 +30,23 @@
 
 __all__ = [
     "Constraint",
+    "DIM",
     "Dim",
     "dims",
     "dynamic_dim",
     "refine_dynamic_shapes_from_suggested_fixes",
 ]
 
 
+class DIM(Enum):
```

Comment:
We have `Dim` already, I was hoping we could hang these constants off that name. I can try in a follow up.

So something like 
```python
import Dim from torch.export
my_tensor_shape = (Dim("batch"), 5, Dim.AUTO)
```